constraint/pkg/client/drivers/k8scel/schema/schema.go

```diff
@@ -15,8 +16,12 @@ import (
 
 const (
 	// Name is the name of the driver.
-	Name           = "K8sNativeValidation"
-	ReservedPrefix = "g8r-"
+	Name = "K8sNativeValidation"
+	// ReservedPrefix signifies a prefix that no user-defined value (variable, matcher, etc.) is allowed to have.
+	// This gives us the ability to add new variables in the future without worrying about breaking pre-existing templates.
+	ReservedPrefix = "g8r_internal_"
```

Comment:
Should we rename this to `gatekeeper_...`? Users/contributors might not know what g8r means? Maybe rename K8s to Kubernetes too?

Reply:
TBH, I'm mostly concerned about brevity/uniqueness. Since this is more about reserving a namespace for us, ideally users never collide with the naming and therefore never need to know the details.

I'm okay with either, but would rather not rename things more than one more time in this PR.

@ritazh preferred name?

Reply:
I'm ok with either as well. +1 on "gatekeeper_" since g8r is less well known.